From: Roman Vasiyarov <rvasiyarov@gmail.com>
Date: Mon, 25 Apr 2011 21:34:23 +0400
Subject: [PATCH] cpufrequtils sysfs: increase MAX_LINE_LEN

larger sysfs data (>255 bytes) was truncated and thus used improperly

Signed-off-by: Roman Vasiyarov <rvasiyarov@gmail.com>
Signed-off-by: Dominik Brodowski <linux@dominikbrodowski.net>

Imported from cpufrequtils_008-1.1.debian.tar.xz

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 lib/sysfs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/sysfs.c b/lib/sysfs.c
index 4e0edabc0e1a..24dd56315abd 100644
--- a/lib/sysfs.c
+++ b/lib/sysfs.c
@@ -18,7 +18,7 @@
 #include "cpufreq.h"
 
 #define PATH_TO_CPU "/sys/devices/system/cpu/"
-#define MAX_LINE_LEN 255
+#define MAX_LINE_LEN 4096
 #define SYSFS_PATH_MAX 255
 
 /* helper function to read file from /sys into given buffer */
